### PR TITLE
Improve CLI help

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: cli/flags.rs [hunk 1]
```diff
@@ -83,88 +98,76 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
...
-type:      type of script (e.g. JavaScript/TypeScript/JSON)
-compiled:  (TypeScript only) shown local path of compiled source code.
-map:       (TypeScript only) shown local path of source map.
-deps:      dependency tree of the source file.
+The following information is shown:
 
-  deno info myfile.ts
-",
+  local: local path of the file.
+  type: JavaScript,TypeScript, or JSON.
```
Comment (bartlomieju) [inline cli/flags.rs]: nit: missing space

File: cli/flags.rs [hunk 2]
```diff
@@ -83,88 +98,76 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
...
+  compiled: TypeScript only. shown local path of compiled source code.
+  map: TypeScript only. shown local path of source map.
+  deps: dependency tree of the source file.",
```
Comment (bartlomieju) [inline cli/flags.rs]: nit: capitalization, also aligned whitespace in previous version was more readable IMHO

File: cli/flags.rs [hunk 3]
```diff
@@ -83,88 +98,76 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
...
+  deno eval 'console.log(\"hello world\")'",
         ).arg(Arg::with_name("code").takes_value(true).required(true)),
     ).subcommand(
       SubCommand::with_name("fmt")
         .setting(AppSettings::DisableVersion)
         .about("Format files")
         .long_about(
-          "
-Format given list of files using Prettier. Automatically downloads
+"Built-in auto formatter (prettier)
```
Comment (bartlomieju) [inline cli/flags.rs]: nit: inconsistent capitalization with sentence below

File: cli/flags.rs (hunk 1 above)
Comment (ry) [reply cli/flags.rs]: fixed

File: cli/flags.rs [hunk 4]
```diff
@@ -41,11 +41,27 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
...
+
+  deno
+
+To execute a sandboxed script:
+
+  deno run https://deno.land/welcome.ts
+
+To evaluate code from the command line:
+
+  deno eval 'console.log(30933 + 404)'
```
Comment (piscisaureus) [inline cli/flags.rs]: Can you use " double quotes? 
That makes it windows compatible.

File: cli/flags.rs [hunk 5]
```diff
@@ -83,88 +99,75 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
...
         .long_about(
-          "
-Evaluate provided script.
+          "Evaluate provided script.
 
 This command has implicit access to all permissions (equivalent to deno run --allow-all)
 
-  deno eval 'console.log(\"hello world\")'
-",
+  deno eval 'console.log(\"hello world\")'",
```
Comment (piscisaureus) [inline cli/flags.rs]: Idem

Comment (piscisaureus) [reply cli/flags.rs]: And then use ' around 'hello world'

Comment (ry) [reply cli/flags.rs]: done

**Pull request merged**